llarp/ev_kqueue.hpp

```diff
@@ -42,7 +42,15 @@ namespace llarp
       sockaddr* addr = (sockaddr*)&src;
       ssize_t ret    = ::recvfrom(fd, buf, sz, 0, addr, &slen);
       if(ret == -1)
+      {
+        llarp::LogWarn("recvfrom failed");
```

Comment:
this should probably not return -1 here, i remember changing this in epoll version.

Reply:
yea, kqueue may need to be updated, I did see you took a pass at syncing in the last week, not sure if it's still needed.